conf/base.config

```diff
@@ -10,37 +10,38 @@
 
 process {
 
-    cpus   = { 1    * task.attempt }
-    memory = { 8.GB * task.attempt }
```

Comment:
revert this entire file, the 8GB/core is optimal for our servers

Reply:
Thanks!